src/main/java/emissary/util/PayloadUtil.java

```diff
@@ -61,22 +90,63 @@ public static String getPayloadDisplayString(final IBaseDataObject payload) {
         final StringBuilder sb = new StringBuilder();
         final List<TransformHistory.History> th = payload.getTransformHistory().getHistory();
         final String fileName = payload.getFilename();
+        final String fileType = payload.getFileType();
         final List<String> currentForms = payload.getAllCurrentForms();
         final Date creationTimestamp = payload.getCreationTimestamp();
 
         sb.append("\n").append("filename: ").append(fileName).append("\n").append("   creationTimestamp: ").append(creationTimestamp).append("\n")
-                .append("   currentForms: ").append(currentForms).append("\n").append("   filetype: ").append(payload.getFileType()).append("\n")
+                .append("   currentForms: ").append(currentForms).append("\n").append("   filetype: ").append(fileType).append("\n")
                 .append("   transform history (").append(th.size()).append(") :").append("\n");
-        for (final TransformHistory.History h : th) {
-            sb.append("     ");
-            if (h.wasCoordinated()) {
-                sb.append("  ");
-            }
-            sb.append(h.getKey()).append("\n");
+
+        // transform history output
+        String historyCase = configureHistoryCase(fileType);
+        switch (historyCase) { // leaving as switch for future transform history altercations
+            case REDUCED_HISTORY:
+                // found reduced history match, output only dropoff
+                sb.append("   ** reduced transform history **").append("\n").append("     dropOff -> ")
+                        .append(payload.getLastPlaceVisited())
+                        .append("\n");
+                break;
+            default:
+                for (final TransformHistory.History h : th) {
+                    sb.append(" ");
+                    if (h.wasCoordinated()) {
+                        sb.append(" ");
+                    }
+
+                    // check is NO_URL or not
+                    if (historyCase.equals(NO_URL)) {
+                        sb.append("    ").append(h.getKeyNoUrl()).append("\n");
+                    } else {
+                        sb.append("    ").append(h.getKey()).append("\n");
+                    }
+                }
         }
         return sb.toString();
     }
 
+    /**
+     * Check if fileType in PayloadUtil.cfg matches current payload fileType
+     *
+     * @param fileType current payload fileType
+     * @return string for output case
+     */
+    public static String configureHistoryCase(String fileType) {
+        for (String currentReducedItem : reducedTransformHistory) {
+            if (!currentReducedItem.equals("") && fileType.equalsIgnoreCase(currentReducedItem)) {
+                return REDUCED_HISTORY;
```

Comment:
What happens if the filetype is in both REDUCED_HISTORY and NO_URL? REDUCED_HISTORY takes precedence, but is that expected? Would we want both? If we don't want both at the same time, I'd update the PayloadUtil.cfg with some explanation on the expected behavior if filetype is in both lists.

Reply:
@cfkoehler what do you think about this? I don't know if your initial intent wanted multiple reduction cases at once or not. Could be something built out later as well

Reply:
My opinion is that a filetype can have only a single explicit "history preference": MINIMAL or NO_URL.  If I were to implement it, I'd use a Map (filetype -> preference_type), and first one in wins.  Would probably log a warning upon conflicting entries for a given filetype.  That's easier for me to wrap my head around and (I think) more easily accommodates future preference options than separate Lists can.

Reply:
Yeah, I like that approach as well.